pkgs/tools/admin/speakeasy-cli/default.nix

```
@@ -0,0 +1,101 @@
+{ pkgs
+, lib
+, stdenv
+, fetchurl
+, unzip
+, nixosTests
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "speakeasy-cli does not support system: ${system}";
+
+  pname = "speakeasy-cli";
+
+  x86_64-darwin-version = "1.128.1";
+  x86_64-darwin-sha256  = "c8f13778e1d03a72e8bc00804129b208706f5a34588735187ea110c7ced90b69";
+
+  x86_64-linux-version = "1.128.1";
+  x86_64-linux-sha256  = "106b0645af2e7ceccfc432205196bf6fff8078ef990932fde59e0ba506101ceb";
+
+  aarch64-darwin-version = "1.128.1";
+  aarch64-darwin-sha256  = "cc5a3fcecba9b6459d850d50d397130a72180679e9d753bc9c5c73a72af93d8b";
+
+  aarch64-linux-version = "1.128.1";
+  aarch64-linux-sha256  = "c2fd0d9ddf95a6206d77ff2680bf571a567799fb39421d010142e03124ffd8ae";
+
+  version = {
+    x86_64-darwin   = x86_64-darwin-version;
+    x86_64-linux    = x86_64-linux-version;
+    aarch64-darwin  =  aarch64-darwin-version;
+    aarch64-linux   =  aarch64-linux-version;
+  }.${system} or throwSystem;
+
+  src = let
+    base = "https://github.com/speakeasy-api/speakeasy/releases/download";
+  in {
+    x86_64-darwin = fetchurl {
+      url    = "${base}/v${version}/speakeasy_darwin_amd64.zip";
+      sha256 = x86_64-darwin-sha256;
+    };
+    x86_64-linux = fetchurl {
+      url    = "${base}/v${version}/speakeasy_linux_amd64.zip";
+      sha256 = x86_64-linux-sha256;
+    };
+    aarch64-darwin = fetchurl {
+      url    = "${base}/v${version}/speakeasy_darwin_arm64.zip";
+      sha256 = aarch64-darwin-sha256;
+    };
+    aarch64-linux = fetchurl {
+      url    = "${base}/v${version}/speakeasy_linux.zip";
+      sha256 = aarch64-darwin-sha256;
+    };
+  }.${system} or throwSystem;
+
+  meta = with lib; {
+    description = "CLI tool for Speakeasy";
+    mainProgram = "speakeasy";
+    changelog   = "https://github.com/speakeasy-api/speakeasy/releases";
+    platforms   = [ "x86_64-darwin" "x86_64-linux" "aarch64-darwin" "aarch64-linux" ];
+    homepage    = "https://www.speakeasyapi.dev/";
+    license     = licenses.free;
```

```suggestion
    license     = licenses.elastic20;
```
This is the correct license.